src/main/java/core/basesyntax/model/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class RightTriangle extends Color {
```

Comment:
The class 'RightTriangle' should not extend 'Color'. In Java, a class should ideally represent a single concept. A triangle is a geometric figure, while color is an attribute of that figure, not a type or behavior. Consider using composition instead of inheritance here, such as having a color property inside the triangle class.